src/lib/CollectStore.js

```diff
@@ -145,7 +145,7 @@ export default class CollectStore {
               arguments: '0 0 0 * * *',
               worker: 'konnector',
               worker_arguments: {
-                konnector: connection.konnector.attributes.slug,
+                konnector: connection.konnector.slug,
```

Comment:
Thanks. Last thing: can we throw an error instead of logging it ? Again, if it implies more dev, let just remove this part for now.